tensorboard/plugins/debugger_v2/debugger_v2_plugin_test.py

```diff
@@ -18,26 +18,107 @@
 from __future__ import division
 from __future__ import print_function
 
+import json
+import os
+import shutil
 import tempfile
 
 import tensorflow as tf
+from werkzeug import test as werkzeug_test  # pylint: disable=wrong-import-order
+from werkzeug import wrappers
 
+from tensorboard.backend import application
 from tensorboard.plugins import base_plugin
 from tensorboard.plugins.debugger_v2 import debugger_v2_plugin
+from tensorboard.util import test_util
 
 
+def _generate_tfdbg_v2_data(logdir):
```

Comment:
Nice; this is clear and thorough.

Reply:
Ack. I don't rule out possible future additions to this method, e.g.,
- Allowing specifying value of the `circular_buffer_size` kwarg to `enable_dump_debug_info()`
- Allowing specifying the `tensor_debug_mode` kwarg to `enable_dump_debug_info()`